bigbluebutton-html5/imports/ui/components/dropdown/component.jsx

```diff
@@ -115,28 +121,27 @@ class Dropdown extends Component {
     const triggerElement = findDOMNode(this.trigger);
     const contentElement = findDOMNode(this.content);
     if (!(triggerElement && contentElement)) return;
-    if (keepOpen === null) {
-      if (triggerElement.contains(event.target)) {
+    if (triggerElement && triggerElement.contains(event.target)) {
+      if (keepOpen) {
+        onHide();
+        return;
+      }
+      if (isOpen) {
+        this.handleHide();
         return;
       }
     }
 
-    if (triggerElement && triggerElement.contains(event.target)) {
-      if (keepOpen) return onHide();    
-      if (isOpen) return this.handleHide();
-    }
-    
     if (keepOpen && isOpen && !contentElement.contains(event.target)) {
```

Comment:
```suggestion
     if (keepOpen && isOpen && triggerElement && !contentElement.contains(event.target)) {
```

Looks like we may have issues with undefined